app/dashboards/sponsor_dashboard.rb

```diff
@@ -61,7 +61,7 @@ class SponsorDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how sponsors are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(sponsor)
-  #   "Sponsor ##{sponsor.id}"
-  # end
+  def display_resource(sponsor)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)